specs/darwin/mdls.table

```diff
@@ -1,7 +1,7 @@
 table_name("mdls")
 description("Query file metadata in the Spotlight database.")
 schema([
-    Column("path", TEXT, "Path of the file", required=True),
+    Column("path", TEXT, "Path of the file", required=True, optimized=True),
```

Comment:
I wonder if https://github.com/osquery/osquery/blob/aa60bbdf2ea8a1ab160f7f1b40db658a8a3770d3/osquery/tables/system/darwin/mdls.mm#L128-L131 is going to tweak this